skorecard/bucketers/bucketers.py

```diff
@@ -455,93 +347,44 @@ def __init__(
         self.remainder = remainder
         self.kwargs = kwargs
 
-    def fit(self, X, y=None):
-        """Fit X, y."""
-        X = self._is_dataframe(X)
-        self.variables = self._check_variables(X, self.variables)
-        self._verify_specials_variables(self.specials, X.columns)
-
-        features_bucket_mapping_ = {}
-        self.binners = {}
-        self.bucket_tables_ = {}
+        self.variables_type = "numerical"
+        self.binners: Dict[str, Any] = {}
```

Comment:
why do we have this for only a few bucketers?

Reply:
see above